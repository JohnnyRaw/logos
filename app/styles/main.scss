// *** Vendor
@import '../../node_modules/sass-mediaqueries/media-queries';

// *** Base
@import 'base/variables';
@import 'base/icons';
@import 'base/typography';

// *** Utilities
@import 'utilities/functions';
@import 'utilities/mixins';
@import 'utilities/grid';

// *** Vendor
@import 'vendor/bootstrap';
@import 'vendor/bootstrap-theme';
@import 'vendor/footable';
@import 'vendor/react-widgets';

// *** Layout
@import 'layout/global';
@import 'layout/app';

// *** Components
@import 'components/form';
@import 'components/footer';
@import 'components/header';
@import 'components/images';
@import 'components/input-switch';
@import 'components/loader';
@import 'components/logo';
@import 'components/toolbar';
@import 'components/system-notifications';

// *** Containers
@import 'containers/cms';
@import 'containers/login';
@import 'containers/items';
@import 'containers/not-found';
