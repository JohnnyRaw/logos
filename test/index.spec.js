import createRoutes from 'routes';
import { ActionTypes, XHR } from 'constants/index';

jest.mock('expose?$!expose?jQuery!jquery', () => {});
jest.mock('bootstrap/js/dist/modal', () => {});
jest.mock('footable/compiled/footable', () => {});

require('main');

describe('Constants:ActionTypes', () => {
  it('should match the snapshot', () => {
    expect(ActionTypes).toMatchSnapshot();
  });
});


describe('Constants:XHR', () => {
  it('should match the snapshot', () => {
    expect(XHR).toMatchSnapshot();
  });
});

describe('Routes', () => {
  it('should match the snapshot', () => {
    const routes = createRoutes();
    expect(routes.props.children.length).toMatchSnapshot();
  });
});
