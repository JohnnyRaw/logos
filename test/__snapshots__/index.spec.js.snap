exports[`Constants:ActionTypes should match the snapshot 1`] = `
Object {
  "CONNECT_CATEGORIES_FAILURE": "CONNECT_CATEGORIES_FAILURE",
  "CONNECT_CATEGORIES_REQUEST": "CONNECT_CATEGORIES_REQUEST",
  "CONNECT_CATEGORIES_SUCCESS": "CONNECT_CATEGORIES_SUCCESS",
  "CONNECT_CATEGORIES_UPDATE": "CONNECT_CATEGORIES_UPDATE",
  "CONNECT_FIREBASE_FAILURE": "CONNECT_FIREBASE_FAILURE",
  "CONNECT_FIREBASE_REQUEST": "CONNECT_FIREBASE_REQUEST",
  "CONNECT_FIREBASE_SUCCESS": "CONNECT_FIREBASE_SUCCESS",
  "CONNECT_LOGOS_FAILURE": "CONNECT_LOGOS_FAILURE",
  "CONNECT_LOGOS_REQUEST": "CONNECT_LOGOS_REQUEST",
  "CONNECT_LOGOS_SUCCESS": "CONNECT_LOGOS_SUCCESS",
  "CONNECT_LOGOS_UPDATE": "CONNECT_LOGOS_UPDATE",
  "CONNECT_TAGS_FAILURE": "CONNECT_TAGS_FAILURE",
  "CONNECT_TAGS_REQUEST": "CONNECT_TAGS_REQUEST",
  "CONNECT_TAGS_SUCCESS": "CONNECT_TAGS_SUCCESS",
  "CONNECT_TAGS_UPDATE": "CONNECT_TAGS_UPDATE",
  "FILTER_ITEMS": "FILTER_ITEMS",
  "HIDE_ALERT": "HIDE_ALERT",
  "SHOW_ALERT": "SHOW_ALERT",
  "UPDATE_CATEGORIES_FAILURE": "UPDATE_CATEGORIES_FAILURE",
  "UPDATE_CATEGORIES_REQUEST": "UPDATE_CATEGORIES_REQUEST",
  "UPDATE_CATEGORIES_SUCCESS": "UPDATE_CATEGORIES_SUCCESS",
  "UPDATE_LOGOS_FAILURE": "UPDATE_LOGOS_FAILURE",
  "UPDATE_LOGOS_REQUEST": "UPDATE_LOGOS_REQUEST",
  "UPDATE_LOGOS_SUCCESS": "UPDATE_LOGOS_SUCCESS",
  "UPDATE_TAGS_FAILURE": "UPDATE_TAGS_FAILURE",
  "UPDATE_TAGS_REQUEST": "UPDATE_TAGS_REQUEST",
  "UPDATE_TAGS_SUCCESS": "UPDATE_TAGS_SUCCESS",
  "USER_LOGIN_FAILURE": "USER_LOGIN_FAILURE",
  "USER_LOGIN_REQUEST": "USER_LOGIN_REQUEST",
  "USER_LOGIN_SUCCESS": "USER_LOGIN_SUCCESS",
  "USER_LOGOUT_FAILURE": "USER_LOGOUT_FAILURE",
  "USER_LOGOUT_REQUEST": "USER_LOGOUT_REQUEST",
  "USER_LOGOUT_SUCCESS": "USER_LOGOUT_SUCCESS",
  "USER_PERMISSIONS": "USER_PERMISSIONS",
  "USER_RESTORE_FAILURE": "USER_RESTORE_FAILURE",
  "USER_RESTORE_REQUEST": "USER_RESTORE_REQUEST",
  "USER_RESTORE_SUCCESS": "USER_RESTORE_SUCCESS"
}
`;

exports[`Constants:XHR should match the snapshot 1`] = `
Object {
  "FAIL": "FAIL",
  "SUCCESS": "SUCCESS"
}
`;

exports[`Routes should match the snapshot 1`] = `3`;
